var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -50,9 +50,13 @@ class Geant4(CMakePackage):
     variant('opengl', default=False, description='Optional OpenGL support')
     variant('x11', default=False, description='Optional X11 support')
     variant('motif', default=False, description='Optional motif support')
+    variant('threads', default=True, description='Build with multithreading')
 
     depends_on('cmake@3.5:', type='build')
 
+    conflicts('+cxx14', when='+cxx11')
+    conflicts('+cxx11', when='+cxx14')
```

Comment:
A single-valued variant would avoid this sort of issue (i.e. have a variant called `std`). Looks like there's some other logic related to dependencies which makes use of this so not important to resolve for this PR